sunpy/instr/rhessi.py

```diff
@@ -348,3 +391,183 @@ def _build_energy_bands(label, bands):
     unit = matched.group('UNIT').strip()
 
     return [f'{band} {unit}' for band in bands]
+
+
+def get_flare_list(start: str,
+                   end: str,
+                   source: str='NASA',
+                   file_format: str="hessi_flare_list_%Y%m.fits",
+                   inc=relativedelta(months=+1)):
```

Comment:
Stupid question, why the +1?

Reply:
Well, I want to iterate through every month... the + is optional, but it seems to be preferred syntax. I guess it's clearer whether you want to increase or decrease a date this way..